lib/vdsm/virt/backup.py

```diff
@@ -352,8 +352,20 @@ def _get_backup_disks(vm, backup_cfg):
 
```

Comment:
@barpavel if this is a WIP, why did you add it to this PR, which should be ready for
review?

You can work on this in another branch. Please add to the PR when its ready
for review.